### PR TITLE
feat: add gitcommit parser

### PR DESCRIPTION
**Pull request by gbprod**
Here is the parser: https://github.com/gbprod/tree-sitter-gitcommit
I use it from days without any problem :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -209,6 +209,7 @@ We are looking for maintainers to add more parsers and to write query files for
 - [x] [fusion](https://gitlab.com/jirgn/tree-sitter-fusion.git) (maintained by @jirgn)
 - [x] [Godot (gdscript)](https://github.com/PrestonKnopp/tree-sitter-gdscript) (maintained by @Shatur95)
 - [x] [git_rebase](https://github.com/the-mikedavis/tree-sitter-git-rebase) (maintained by @gbprod)
+- [x] [gitcommit](https://github.com/gbprod/tree-sitter-gitcommit) (maintained by @gbprod)
 - [x] [gitattributes](https://github.com/ObserverOfTime/tree-sitter-gitattributes) (maintained by @ObserverOfTime)
```
Comment (ObserverOfTime) [inline README.md:213]:
```suggestion
- [x] [gitattributes](https://github.com/ObserverOfTime/tree-sitter-gitattributes) (maintained by @ObserverOfTime)
- [x] [gitcommit](https://github.com/gbprod/tree-sitter-gitcommit) (maintained by @gbprod)
```

File: queries/gitcommit/highlights.scm [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    ")"
+    ":"
+] @punctuation.delimiter)
+(prefix [
+    "!"
+] @punctuation.special)
+
+(message) @text
+
+(trailer (token) @keyword)
```
Comment (ObserverOfTime) [inline queries/gitcommit/highlights.scm:25]: `@label` might be better.
```suggestion
(trailer (token) @label)
```

**Review by ObserverOfTime — CHANGES_REQUESTED** [review]

**Comment by gbprod** [comment]
Thanks for your feedbacks @ObserverOfTime :)

**Review by theHamsta — APPROVED** [review]
Thank you!

**Pull request merged**